cmd/create/cluster/cmd.go

```diff
@@ -598,6 +611,20 @@ func run(cmd *cobra.Command, _ []string) {
 		isIAM = !isSTS
 	}
 
+	isHostedCluster := args.hostedClusterEnabled
+	if interactive.Enabled() && cmd.Flags().Changed("hosted-cp") {
+		isHostedCluster, err = interactive.GetBool(interactive.Input{
```

Comment:
/nitpick
Variable name should probably be `isHostedCP`

Reply:
done